Dockerfile.assisted-service

```diff
@@ -43,9 +43,35 @@ RUN cd ./cmd/operator && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /bu
 RUN cd ./cmd/webadmission && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service-admission
 RUN cd ./cmd/agentbasedinstaller/client && CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/agent-installer-client
 
+# Licenses
+FROM registry.access.redhat.com/ubi9/go-toolset:1.18 AS licenses
+
+ADD . /app
+WORKDIR /app
+
+RUN go install github.com/google/go-licenses@v1.6.0
+RUN ${HOME}/go/bin/go-licenses save --save_path /tmp/licenses ./...
```

Comment:
nit: should we use only one `RUN` directive to reduce number of layers?

Reply:
even for a intermediate stage ?

I think it can also be done for the LABELs